modules/reactor/src/meshgenerators/PinMeshGenerator.C

```diff
@@ -309,8 +293,6 @@ PinMeshGenerator::PinMeshGenerator(const InputParameters & parameters)
     for (int i = 0; i < num_sides; i++)
       boundaries_to_delete.insert(boundaries_to_delete.end(),
                                   {std::to_string(10001 + i), std::to_string(15001 + i)});
-    if (num_sides == 4)
-      boundaries_to_delete.push_back(std::to_string(20000 + _pin_type));
```

Comment:
where is this done now

Reply:
So before, we creating this boundary in PolygonConcentricCircleMeshGenerator, then rotating the pin and then assigning a new boundary to the rotated pin since the boundary wasn't propagating forward. Therefore, we had to delete the 20000 + _pin_type boundary that was created in the first step. Now, however, a lot of this logic is consolidated into a single MG using flat_side_up=true (new functionality in PCCMG), so the need to create a new boundary and delete the old 20000 + _pin_type boundary is no longer needed. We now just use the 20000 + _pin_type boundary directly

Reply:
but you dont want to output this boundary in the final mesh right

Reply:
Yes let's remove extra sidesets. The mesh should be as clean as possible coming out at any stage of the workflow.

Currently what stages have to have extra sidesets for future stitching?